contracts/staking/PoSValidatorManager.sol

```diff
@@ -158,29 +156,32 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         require(
             validator.status == ValidatorStatus.Active, "PoSValidatorManager: validator not active"
         );
+        // Update the validator weight
+        uint64 newValidatorWeight = validator.weight + weight;
+        _setValidatorWeight(validationID, newValidatorWeight);
+
+        uint64 nonce = _getAndIncrementNonce(validationID);
+        bytes32 delegationID = sha256(abi.encodePacked(validationID, delegatorAddress, nonce));
```

Comment:
I think we've used keccack for hashing in all places we're able to since it's slightly less gas and arguably more secure (though don't think that matters in this case here).

Should we use keccack here also just for consistency though?

Reply:
I chose `sha256` to be consistent with `validationID`, but don't feel strongly either way. Since the `delegationID` isn't used outside of the validator manager contracts, I think gas cost should be the primary consideration.